types/wlr_screencopy_v1.c

```diff
@@ -132,7 +134,8 @@ static void frame_destroy(struct wlr_screencopy_frame_v1 *frame) {
 	if (frame == NULL) {
 		return;
 	}
-	if (frame->output != NULL && frame->buffer != NULL) {
+	if (frame->output != NULL && frame->shm_buffer != NULL &&
+			frame->dma_buffer) {
```

Comment:
This condition will never be true. Maybe the end of it should be changed to `(frame->shm_buffer != NULL || frame->dma_buffer != NULL)`?

Reply:
Yes, that is most likely what I meant to say.